Popup/tests/Popup-specs.js

```diff
@@ -1,236 +1,309 @@
 import {FloatingLayerDecorator} from '@enact/ui/FloatingLayer';
-import {mount, shallow} from 'enzyme';
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
 
-import {Popup, PopupBase} from '../Popup';
-import css from '../Popup.module.less';
+import {Popup} from '../Popup';
 
 const FloatingLayerController = FloatingLayerDecorator('div');
 
 describe('Popup specs', () => {
-	it('should be rendered opened if open is set to true', () => {
-		const popup = mount(
+	test('should be rendered opened if open is set to true', () => {
+		const {getByText} = render(
```

Comment:
🤔 I was curious about this pattern and I found the article that said about this is not recommended. https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen.
In the article, the author said it's recommended to use `screen` instead of this pattern.
What do you think?

Reply:
Thank you so much for the article! Made us reconsider some things and we will proceed to some refactoring according to those recommendations.